airflow/models.py

```diff
@@ -3901,7 +3902,7 @@ def __repr__(self):
         return self.label
 
 
-class Variable(Base):
+class Variable(Base, LoggingMixin):
```

Comment:
Why this is needed?

Reply:
@skudriashev you may need to rebase your branch, this is already in master.

Reply:
@edgarRd sorry?

Reply:
Oh, I'm sorry @skudriashev, wrong tags

Reply:
@GeorgeSirois ^

Reply:
@edgarRd @skudriashev yeah, this particular change was outdated and no longer needed after rebasing.